python/ray/dashboard/client/src/pages/dashboard/logical-view/ActorDetailsPane.tsx

```diff
@@ -48,7 +48,12 @@ const ActorDetailsPane: React.FC<ActorDetailsPaneProps> = ({
           ({ label, value, tooltip }) =>
             value &&
             value.length > 0 && (
-              <LabeledDatum label={label} datum={value} tooltip={tooltip} />
+              <LabeledDatum
+                key={label}
```

Comment:
Can't this cause trouble when there are the same name actors? What does label stand for in this case?

Reply:
A labeled datum is something like
`Actor title:  Foo` where `Actor title` is the label and `Foo` is the value. There isn't a collision in keys because the set of labeled data for each actor is rendered in a different sub-tree of the DOM.